docs/setup/local_setup.md

```diff
@@ -20,11 +20,11 @@ specific language governing permissions and limitations
 under the License.
 -->
 
+This documentation is intended to provide instructions on how to run Flink locally on a single machine.
+
 * This will be replaced by the TOC
 {:toc}
 
-This documentation is intended to provide instructions on how to run Flink locally on a single machine.
-
 ## Download
 
 Go to the [downloads page]({{site.baseurl}}/downloads.html) and get the ready to run package. If you want to interact with Hadoop (e.g. HDFS or HBase), make sure to pick the Flink package **matching your Hadoop version**. When in doubt or you plan to just work with the local file system pick the package for Hadoop 1.2.x.
```

Comment:
The link to the "downloads page" is broken. It will point to http://ci.apache.org/projects/flink/flink-docs-master/downloads.html which doesn't exist.